webroot/mysite/code/Estimate.php

```diff
@@ -195,7 +195,19 @@ private function _getConfidenceLevel($idx){
         return array_key_exists($idx, $this->_confidenceLevels) ? $this->_confidenceLevels[$idx] : '';
     }
 
-    public function GetCssClasses(){
```

Comment:
@complexthings lets discuss a refactor for this. I am thinking a general translation class that handles this type of presentational logic. Come by when you get a few.